Library/Homebrew/extend/pathname.rb

```diff
@@ -289,14 +289,14 @@ def write_exec_script(*targets)
   end
 
   # Writes an exec script that sets environment variables.
-  sig { params(target: Pathname, args: T.any(T::Array[String], T::Hash[String, String]), env: T.nilable(T::Hash[String, String])).void }
+  sig { params(target: Pathname, args: T.any(String, T::Array[String], T::Hash[String, String]), env: T.nilable(T::Hash[String, String])).void }
```

Comment:
Technically these hashes should be
```ruby
T::Hash[T.any(Symbol, String), String]
```
but Sorbet seems rather permissive about this right now, since an `env` could be, say `{ PATH: "/foo:/bar" }`. That might change though.

Reply:
Ah, this is because Sorbet doesn't actually typecheck the types of the keys and values of a hash: https://sorbet.run/#%23%20typed%3A%20true%0Aextend%20T%3A%3ASig%0A%0Asig%20%7Bparams%28x%3A%20T%3A%3AHash%5BT%3A%3ABoolean%2C%20T%3A%3ABoolean%5D%29.void%7D%0Adef%20foo%28x%29%0A%20%20puts%28x%29%0Aend%0A%0Afoo%28%7B%20bar%3A%20%22baz%22%20%7D%29